### ISSUE TITLE
Broken on mac

### ISSUE THREAD

**Issue by pmudry** [hevs-isi/gdx2d#118]
Shaders complain that version 130 is not supported. I had the impression it had been fixed before...

**Comment by pmudry** [comment]
Was fixed in 540cbd699f661f5f7df06b73adc540cdc75e196e
However the student was probably not using the latest code but the release one.

**Comment by metc** [comment]
This is how the demo selector looks like:
![screen shot 2016-04-22 at 14 11 34](https://cloud.githubusercontent.com/assets/229670/14741095/2dceb736-0894-11e6-8445-6bfb16cce48f.png)

Tested on the Pierre's Mac OS X El Capitan (10.11.4)

**Comment by metc** [comment]
The Swing layout is now fixed, but the canvas is still not working on Mac :(

**Comment by pmudry** [comment]
No idea what to do as I don't have any Mac to test. This is probably hard to fix.

**Comment by metc** [comment]
Swing seems to not be very platform independent :( This was the first layout issue. For the second, it will be hard to solve it, yes. The OpenGL canvas is not display on Mac (tested on OS X `10.11.3` latest).

**Comment by metc** [comment]
I think we should set this issue for a next milestone. A Mac guy should help us to solve this. I tried a quick fix but I did not get better results.

Currently, a Mac user can launch every demo individually, but not using the demo selector.

**Comment by pmudry** [comment]
I agree. The problem being mac specific, we can't solve it without a Mac.

**Comment by acely** [comment]
Just providing some information,might help.

Im a mac user,cant run the demo too.
When I use JDK8 (oracle jdk) ,the window looks like below.The canvas was misplaced,and only display grey.
<img width="1010" alt="qq20170102-0 2x" src="https://cloud.githubusercontent.com/assets/5640723/21583640/e78938ba-d0c4-11e6-8a66-26a85a811561.png">

As we know,apple only provide java up to java6,so any later jdk was provided by oracle.And oracle jdk removed all heavyweight components(even awt turned into lightweight),so anything rely on heavyweight component wont work under oracle jdk on mac(like LwjglAWTCanvas). 

So I change the project to apple jdk6(which still have heavyweight components),thinking this may solve the problem.But after I run,I get this: 

<img width="1010" alt="qq20170102-1 2x" src="https://cloud.githubusercontent.com/assets/5640723/21583639/e787991a-d0c4-11e6-91e8-5d9aeb17cb01.png">

The canvas was not misplaced,but still shows nothing.Which makes me really confused.
The console shows the following info when using jdk6:
```
2017-01-02 08:33:43.665 java[17091:87589438] 08:33:43.664 WARNING:  140: This application, or a library it uses, is using the deprecated Carbon Component Manager for hosting Audio Units. Support for this will be removed in a future release. Also, this makes the host incompatible with version 3 audio units. Please transition to the API's in AudioComponent.h.
[gdx2d]: gdx2d-core v1.2.1, libgdx v1.5.6 | mui, chn, mei, pim (c) 2012-2016
[gdx2d]: Fragment shader 'res/lib/circle_aa.fp' loaded.
[gdx2d]: Vertex shader 'res/lib/default.vs' loaded.
[gdx2d]: Shader message - WARNING: Output of vertex shader 'v_color' not read by fragment shader
WARNING: Output of vertex shader 'vTexCoord' not read by fragment shader
WARNING: Output of vertex shader 'vSurfacePosition' not read by fragment shader

```

The console shows the following info when using jdk8:
```
2017-01-02 08:38:05.907 java[17127:87627414] 08:38:05.907 WARNING:  140: This application, or a library it uses, is using the deprecated Carbon Component Manager for hosting Audio Units. Support for this will be removed in a future release. Also, this makes the host incompatible with version 3 audio units. Please transition to the API's in AudioComponent.h.
[gdx2d]: gdx2d-core v1.2.1, libgdx v1.5.6 | mui, chn, mei, pim (c) 2012-2016
[gdx2d]: Fragment shader 'res/lib/circle_aa.fp' loaded.
[gdx2d]: Vertex shader 'res/lib/default.vs' loaded.
[gdx2d]: Shader message - WARNING: Output of vertex shader 'v_color' not read by fragment shader
WARNING: Output of vertex shader 'vTexCoord' not read by fragment shader
WARNING: Output of vertex shader 'vSurfacePosition' not read by fragment shader

```
